src/test/java/org/jenkinsci/plugins/buildnamesetter/BuildNameSetterTest.java

```diff
@@ -2,11 +2,14 @@
 
 import static org.junit.Assert.assertEquals;
 
-import hudson.EnvVars;
-import hudson.model.*;
-
 import java.io.IOException;
 import java.util.concurrent.ExecutionException;
+
+import hudson.EnvVars;
+import hudson.model.FreeStyleBuild;

```

Comment:
Could you please explain why other models are removed and only keep FreeStyleBuild in the latest version? Does it mean that this plugin could only support FreeStyleBuild?

Reply:
I'm not sure - you can try